web/src/features/charts/tooltipCalculations.test.ts

```diff
@@ -119,22 +119,20 @@ describe('getProductionTooltipData', () => {
       Mode.CONSUMPTION
     );
 
-    expect(actual).toMatchInlineSnapshot(`
-      {
-        "capacity": 61370,
-        "co2Intensity": 5.13,
-        "co2IntensitySource": "UNECE 2022",
-        "displayByEmissions": false,
-        "emissions": 211155930,
-        "isExport": false,
-        "production": 41161,
-        "storage": undefined,
-        "totalElectricity": 84545.75,
-        "totalEmissions": 9590208033016.975,
-        "usage": 41161,
-        "zoneKey": "FR",
-      }
-    `);
+    expect(actual).to.deep.equal({
```

Comment:
Does this mean we'll lose the ability to update snapshots?

Reply:
Oh didn't think about that, vitest also has snapshots so I'll just add it as an import and revert to the snapshots.